com.gitlab.cunidev.Gestures.json

```diff
@@ -0,0 +1,41 @@
+{
+    "app-id" : "com.gitlab.cunidev.Gestures",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "40",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "gestures",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=~/.config",
+        "--filesystem=xdg-run/gvfs",
+        "--socket=session-bus"
```

Comment:
Same question as above.

Reply:
Session bus is for flatpak-spawn, that is controlling the libinput-gestures service through its CLI

Reply:
Only `--talk-name=org.freedesktop.Flatpak` is needed for flatpak-spawn.

Reply:
Thank you